src/Term/TermList.php

```diff
@@ -66,7 +67,7 @@ public function toTextArray() {
 
 	/**
 	 * @see IteratorAggregate::getIterator
-	 * @return Traversable|Term[]
+	 * @return Traversable|Iterator|Term[]
```

Comment:
I'm fine with stating Iterator here. I remember when I had to decide what to use, and I went for the more narrow Traversable because that's all that is needed in a foreach loop.

Note that the parent's documentation says "Iterator or Traversable".

Since Iterator extends Traversable, Traversable should be removed here. :+1: otherwise.